be/src/olap/delta_writer.cpp

```diff
@@ -170,14 +170,14 @@ Status DeltaWriter::write(const vectorized::Block* block, const std::vector<int>
     _total_received_rows += row_idxs.size();
     _mem_table->insert(block, row_idxs);
 
-    if (_mem_table->need_to_agg()) {
+    if (_mem_table->need_agg()) {
```

Comment:
Suggest to add `UNLIKELY` when calling `need_agg` and `need_flush` , since the size of a block is much smaller than 200MB.

Reply:
ok